components/input/Search.tsx

```diff
@@ -18,6 +18,7 @@ export interface SearchProps extends InputProps {
       | React.ChangeEvent<HTMLInputElement>
       | React.MouseEvent<HTMLElement>
       | React.KeyboardEvent<HTMLInputElement>,
+    source?: 'clear' | 'input',
```

Comment:
```suggestion
    info: { source?: 'clear' | 'input' },
```
参考其他的 source 应该都是收在对象里的